prettier.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  printWidth: 80,
+  singleQuote: false,
+  tabWidth: 2,
+  semi: true,
+  trailingComma: "all",
```

Comment:
```suggestion
  trailingComma: "es5",
```

I prefer the `es5` style here.

Reply:
Of cause I am down for any config, I don't think I realized the config is supposed to only specify the overrides.